components/tag/style/index.less

```
@@ -119,6 +119,12 @@
   .make-status-color-classes('cyan', processing);
   .make-status-color-classes('red', error);
   .make-status-color-classes('orange', warning);
+
+  // To ensure that a space will be placed between character and `Icon`.
+  > .@{iconfont-css-prefix} + span,
+  > span + .@{iconfont-css-prefix} {
+    margin-left: 8px;
```

```suggestion
    margin-left: 7px;
```

same as Tag padding.